### ISSUE TITLE
Ignition getting ENETUNREACH on Exoscale when not providing user-data

### ISSUE THREAD

**Issue by jlebon** [coreos/fedora-coreos-tracker#436]
This is on the latest testing-devel with NM-in-initrd. Just going to paste IRC logs here:

```
10:15:29 < jlebon> so one thing related here is:
                   https://github.com/coreos/ignition/issues/950
10:15:53 < jlebon> looking at the streaming VGA console logs carefully it does look like
                   the fetch and disk stage reach the metadata server fine
10:16:06 < jlebon> but by the time the files stage runs, it no longer can
10:16:47 < jlebon> NM should've run upfront before fetch though, so the networking setup
                   shouldn't have really changed
10:17:56 < jlebon> which makes me wonder if the platform is somehow not liking ignition
                   spamming the metadata server repeatedly and shutting down its network
                   or something?
10:20:12 < jlebon> IOW, fixing https://github.com/coreos/ignition/issues/950 would've
                   worked around this, but it'd be good to figure out what's really
                   going on there
```

We're trying to get serial console logs for more analysis.

**Comment by pierre-emmanuelJ** [comment]
👍

**Comment by dustymabe** [comment]
this is most likely the same or similar to #440

**Comment by dustymabe** [comment]
@pierre-emmanuelJ - can you try out [31.20200323.2.1](https://builds.coreos.fedoraproject.org/prod/streams/testing/builds/31.20200323.2.1/x86_64/fedora-coreos-31.20200323.2.1-exoscale.x86_64.qcow2.xz) and see if it works?

**Comment by pierre-emmanuelJ** [comment]
Yes thank you! This built fix the ignition issue

**Comment by dustymabe** [comment]
this should be fixed in the latest testing release[`31.20200323.2.1`](https://github.com/coreos/fedora-coreos-streams/issues/76)

**Comment by dustymabe** [comment]
The fix for this went into stable stream release `31.20200323.3.2`.
